sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4050,26 +4043,6 @@ _ZNK2cl4sycl6streameqERKS1_
 _ZNK2cl4sycl6streamneERKS1_
 _ZNK2cl4sycl7context11get_devicesEv
 _ZNK2cl4sycl7context12get_platformEv
-_ZNK2cl4sycl7context12get_propertyINS0_3ext6oneapi8property6buffer22use_pinned_host_memoryEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property5image12use_host_ptrEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property5image13context_boundEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property5image9use_mutexEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property5queue8in_orderEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property6buffer12use_host_ptrEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property6buffer13context_boundEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property6buffer9use_mutexEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property6noinitEEET_v
-_ZNK2cl4sycl7context12get_propertyINS0_8property7context4cuda19use_primary_contextEEET_v
-_ZNK2cl4sycl7context12has_propertyINS0_3ext6oneapi8property6buffer22use_pinned_host_memoryEEEbv
-_ZNK2cl4sycl7context12has_propertyINS0_8property5image12use_host_ptrEEEbv
-_ZNK2cl4sycl7context12has_propertyINS0_8property5image13context_boundEEEbv
-_ZNK2cl4sycl7context12has_propertyINS0_8property5image9use_mutexEEEbv
-_ZNK2cl4sycl7context12has_propertyINS0_8property5queue8in_orderEEEbv
```

Comment:
This line corresponds to `bool cl::sycl::context::has_property<cl::sycl::property::queue::in_order>() const`. I don't see it being changed in patch. Did you accidentally run script against the wrong so file?

Reply:
I merged and regenerated the ABI dump.  It should've only added 3 symbols.

Reply:
Anyone have a clue what's going on with the ABI stuff?  I regenerated according to the instructions and something apparently isn't in order.

Reply:
@alexbatashev  Could you please help?

Reply:
I think the new re-generated file is fine, actually.